app/src/main/java/space/taran/arknavigator/utils/FileUtils.kt

```diff
@@ -89,3 +90,7 @@ fun reifySorting(sorting: Sorting): Comparator<ResourceMeta>? =
 
 const val KILOBYTE = 1024
 const val MEGABYTE = 1024 * KILOBYTE
+
+fun getMimeTypeUsingTika(path: Path): String? {
+    return Tika().detect(Files.newInputStream(path,), "image")
```

Comment:
This code can be important for performance. Is content of the file being read completely to the end or just first bytes?

Reply:
![Screenshot from 2022-06-17 14-19-48](https://user-images.githubusercontent.com/74892023/174263401-ad7f6003-6f71-47cd-800c-076a49fa5531.png)
Looks like the performance is managed by Tika.